doc/sphinx/source/introduction.rst

```diff
@@ -0,0 +1,199 @@
+assessment report, such as Chapter 9 of the Intergovernmental Panel on
+Climate Change (IPCC) Fifth Assessment Report (AR5) (Flato et al.,
+2013). The expectation is that in this way a routine and systematic
+evaluation of model results can be made more efficient, thereby enabling
+scientists to focus on developing more innovative methods of analysis
+rather than constantly having to "reinvent the wheel".
+
+In parallel to standardization of model output, the ESGF also hosts
+observations for Model Intercomparison Projects (obs4MIPs) and
+reanalyses data (ana4MIPs). obs4MIPs provides open access data sets of
+satellite data that are comparable in terms of variables, temporal and
+spatial frequency, and periods to CMIP model output (Taylor et al.,
+2012). The ESMValTool utilizes these observations and reanalyses from
+ana4MIPs plus additionally available observations in order to evaluate
+the models performance. In many diagnostics and metrics, more than one
+observational data set or meteorological reanalysis is used to assess
+uncertainties in observations.
+
+The main idea of the ESMValTool is to provide a broad suite of
+diagnostics which can be performed easily when new model simulations are
+run. The suite of diagnostics needs to be broad enough to reflect the
+diversity and complexity of Earth System Models, but must also be robust
+enough to be run routinely or semi-operationally. In order the address
+these challenging objectives the ESMValTool is conceived as a framework
+which allows community contributions to be bound into a coherent
+framework.
+
+License
+=======
+
+The ESMValTool is released under the Apache License, version 2.0.
+Citation of the ESMValTool paper ("Software Documentation Paper") is
+kindly requested upon use, alongside with the software DOI for
+ESMValTool
+(`doi:10.5281/zenodo.3401363 <https://doi.org/10.5281/zenodo.3401363>`__)
+and ESMValCore
+(`doi:10.5281/zenodo.3387139 <https://doi.org/10.5281/zenodo.3387139>`__)
+and version number:
+
+-  Righi, M., Andela, B., Eyring, V., Lauer, A., Predoi, V., Schlund,
+   M., Vegas-Regidor, J., Bock, L., Brötz, B., de Mora, L., Diblen, F.,
+   Dreyer, L., Drost, N., Earnshaw, P., Hassler, B., Koldunov, N.,
+   Little, B., Loosveldt Tomas, S., and Zimmermann, K.: ESMValTool v2.0
+   - Technical overview, Geosci. Model Dev. Discuss.,
+   https://doi.org/10.5194/gmd-2019-226, in review, 2019.
+
+Besides the above citation, users are kindly asked to register any
+journal articles (or other scientific documents) that use the software
+at the ESMValTool webpage (http://www.esmvaltool.org/). Citing the
+Software Documentation Paper and registering your paper(s) will serve to
+document the scientific impact of the Software, which is of vital
+importance for securing future funding. You should consider this an
+obligation if you have taken advantage of the ESMValTool, which
+represents the end product of considerable effort by the development
+team.
+
+Workflow
```

Comment:
This header perhaps could use a new name. "Workflow" is a bit vague

Reply:
"Basic Workflow when using ESMValTool"?

Reply:
Improved the title